Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -350,7 +350,7 @@ void ClearUnusedPixelShaderUidBits(APIType ApiType, const ShaderHostConfig& host
 
   // If bounding box is enabled when a UID cache is created, then later disabled, we shouldn't
   // emit the bounding box portion of the shader.
-  uid_data->bounding_box &= host_config.bounding_box;
+  uid_data->bounding_box &= host_config.bounding_box & host_config.backend_bbox;
```

Comment:
Not strictly necessary, as bounding box should be force-disabled if the backend does not support it at startup time. But doesn't hurt to mask it anyway.

Reply:
uid cache will be used in different backend, and it should be cleared if backend doesn't support.

Reply:
Hmm, looks like I was incorrect about it being force disabled. Probably should do that in VideoConfig::CheckValidity(), but having the check here does not hurt.